services/workflows-service/src/customer/customer.controller.external.ts

```diff
@@ -49,14 +55,17 @@ export class CustomerControllerExternal {
   @common.Get('/by-current-project-id')
   @swagger.ApiOkResponse({ type: [CustomerModel] })
   @swagger.ApiForbiddenResponse()
-  async getByCurrentProjectId(
-    @CurrentProject() currentProjectId: TProjectId,
-  ): Promise<TCustomerWithFeatures | null> {
+  async getByCurrentProjectId(@CurrentProject() currentProjectId: TProjectId): Promise<
```

Comment:
Wouldn't something like this also work ?
```suggestion
  async getByCurrentProjectId(@CurrentProject() currentProjectId: TProjectId): Promise<(TCustomerWithFeatures & { config: { demoAccessDetails: TDemoAccessDetails } } ) | null>
```

In either case I would create separate type for TCustomerWithFeatures with demo access details so the return type isn't as messy.